schema/mysql/v8/temporal/schema.sql

```diff
@@ -336,3 +336,19 @@ CREATE TABLE queue_messages (
         message_id
     )
 );
+
+-- Stores information about Nexus incoming services
+CREATE TABLE nexus_incoming_services (
+    service_id      BINARY(16) NOT NULL,
+    data            MEDIUMBLOB NOT NULL,  -- temporal.server.api.persistence.v1.NexusIncomingService
+    data_encoding   VARCHAR(16) NOT NULL, -- Encoding type used for serialization, in practice this should always be proto3
+    version         BIGINT NOT NULL,      -- Version of this row, used for optimistic concurrency
+    PRIMARY KEY (service_id)
+);
+
+-- Stores the version of Nexus incoming services table as a whole
+CREATE TABLE nexus_incoming_services_table_version (
+    service_type    ENUM('incoming') NOT NULL,  -- Using an ENUM with a single value as the primary key ensures this table will only ever have one row
```

Comment:
I would just use an int ID as the primary key and always use ID 0 in the queries in code to avoid the discrepancies between the SQL implementations.